Demo/RCHSearchViewController.swift

```diff
@@ -170,18 +205,18 @@ class RCHSearchViewController: UIViewController, UISearchBarDelegate, UICollecti
         let autocompleteString = autocompleteArray[indexPath.row]
 
         if !searchTerm.isEmpty && autocompleteArray.count > 0 {
-            let searchTermLength = searchTerm.characters.count
-            let autocompleteTermLength = autocompleteString.characters.count
-            
-            if autocompleteTermLength >= searchTermLength {
+                let searchString = searchTerm.lowercased()
                 let highlightColor = UIColor(red: 0, green: 121/255, blue: 253/255, alpha: 1)
                 let blueAttribute = [NSBackgroundColorAttributeName : highlightColor]
-                
-                let attributedString = NSMutableAttributedString(string: autocompleteString, attributes: nil)
-                attributedString.addAttributes(blueAttribute, range: NSRange.init(location: 0, length: searchTermLength))
-                
+                let attributedString = NSMutableAttributedString(string: autocompleteString)
+            
+                let range: Range<String.Index> = autocompleteString.range(of: searchString)!
```

Comment:
Typing fast seems to crash it. I think `searchString` is updated faster than the response.